conf/base.config

```diff
@@ -85,4 +85,9 @@ process {
         memory = { 4.GB * task.attempt }
         time   = { 4.h  * task.attempt }
     }
+
+    withName: SYLPHTAX_TAXPROF  {
+        errorStrategy = 'ignore'
```

Comment:
What is the reason for this?

Reply:
I am keeping this as discussed in slack.